src/cascadia/TerminalControl/EventArgs.idl

```diff
@@ -22,6 +22,11 @@ namespace Microsoft.Terminal.Control
         Windows.Foundation.IReference<CopyFormat> Formats { get; };
     }
 
+    runtimeclass ContextMenuRequestedEventArgs
+    {
+        Windows.Foundation.Point Point { get; };

```

Comment:
this is such a nit, but I don't like that it's a `Point Point`. Maybe rename it to `Position`?